src/test/kotlin/no/nav/syfo/services/TssServiceSpek.kt

```diff
@@ -0,0 +1,93 @@
+package no.nav.syfo.services
+
+import io.mockk.clearMocks
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.mockk
+import kotlinx.coroutines.runBlocking
+import no.nav.helse.eiFellesformat2.XMLEIFellesformat
+import no.nav.helse.eiFellesformat2.XMLMottakenhetBlokk
+import no.nav.helse.msgHead.XMLMsgHead
+import no.nav.syfo.UserConstants.LEGEKONTOR_ORGNAME
+import no.nav.syfo.client.SmtssClient
+import no.nav.syfo.client.TssIdent
+import no.nav.syfo.client.TssIdSource
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.util.fellesformatUnmarshaller
+import no.nav.syfo.util.get
+import no.nav.syfo.util.getFileAsString
+import org.amshove.kluent.shouldBeEqualTo
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.io.StringReader
+
+class TssServiceSpek : Spek({
+    describe("TssService") {
+        val smtssClient = mockk<SmtssClient>(relaxed = true)
+        val tssService = TssService(smtssClient = smtssClient)
+
+        val felleformat = fellesformatUnmarshaller.unmarshal(
+            StringReader(getFileAsString("src/test/resources/dialogmelding_dialog_svar_foresporsel_om_pasient.xml"))
+        ) as XMLEIFellesformat
+
+        val msgHead: XMLMsgHead = felleformat.get()
+        val receiverBlock = felleformat.get<XMLMottakenhetBlokk>()
+        val personNumberDoctor = receiverBlock.avsenderFnrFraDigSignatur
+        val legePersonIdent = PersonIdent(personNumberDoctor)
+        val msgId = msgHead.msgInfo.msgId
+
+        beforeEachTest {
+            clearMocks(smtssClient)
+        }
```

Comment:
Siden disse er innkapslet i describen, vil de ikke kjøre hvis man lager en ny describe nå?
Ser at noen stedet står dette utenfor den første describen, andre stedet inni. Hva er "riktigst"?

Reply:
Nå er det bare én describe, så da tenker vi den bare kan være der den er nå, i stedet for å gjøre en endring fordi det kanskje kommer flere describes senere 👍